### DISCUSSION TITLE
Duration returns 0 for mp3 from sourceurl

### DISCUSSION THREAD

**Discussion by DonChen1** [bluefireteam/audioplayers#1602] [Q&A]
Any idea how to get this working? Duration returns 0 for mp3 frile played from a url. i am using the duration value for my slider.

**Comment by Gustl22** [comment]
See player widget for an implementation: https://github.com/bluefireteam/audioplayers/blob/main/packages/audioplayers/example/lib/components/player_widget.dart

Which platform are you using? On Linux, the platform implementation has problems finding out the duration for MP3 files with variable bit rate (VBR). Maybe this is the reason. Best regards.
